.gitmodules

```diff
@@ -974,3 +974,9 @@
 [submodule "libraries/drivers/cst8xx"]
 	path = libraries/drivers/cst8xx
 	url = https://github.com/adafruit/Adafruit_CircuitPython_CST8XX.git
+[submodule "libraries/drivers/vcnl4020"]
+	path = libraries/drivers/vcnl4020
+	url = https://github.com/adafruit/Adafruit_CircuitPython_VCNL4020.git
+[submodule "libraries/drivers/libraries/drivers/vcnl4020"]
```

Comment:
It looks like you accidentally left a duplicate submodule.

Reply:
thanks!